frontend/src/store.js

```diff
@@ -0,0 +1,16 @@
+import { createStore, compose, applyMiddleware, combineReducers } from 'redux';
```

Comment:
this method is depreciated. you have to import createStore like this 
`import { configureStore } from '@reduxjs/toolkit';` separately